boltdesignsystem/bolt
# Fix: Adds support for ID attribute to button component

## Jira

http://vjira2:8080/browse/BDS-1762

## Summary

Adds support for ID attribute to button element

## How to test

- On master, create a button using the following twig and confirm that it is missing an ID attribute:
```
{% include "@bolt-components-button/button.twig" with {
  text: "Button Test -- ID attribute",
  attributes: {
    id: "my-button"
   }
} only %}
```
- On this branch, confirm that the same button _does_ have an ID (or take the jest test's word for it!)
